build.gradle.kts

```diff
@@ -181,6 +181,8 @@ dependencies {
   }
   implementation("com.azure:azure-messaging-eventhubs")
   implementation("com.azure:azure-identity")
+  implementation("com.azure.resourcemanager:azure-resourcemanager:2.27.0")
```

Comment:
`azure-resourcemanager` seems to not be used in you PR. Could you please check if it's really needed?